### DISCUSSION TITLE
My application education pack was rejected.

### DISCUSSION THREAD

**Discussion by SkajBlock** [community/community#108403] [GitHub Education]
### Hi everyone,

Hello, I am trying to access Github education but it shows me this error: (under the message) All the data is identical on the school card and on the GitHub site, how fix this.

It's error:
 - Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

 - Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information. Variations of your name and nicknames are unacceptable.

**Comment by lunasoldev** [comment]
Have you tried shooting another photo? Sometimes the text reading can be a bit weird.

**Reply by SkajBlock** [reply]
Yes, I took a picture and most likely it is a problem from the side as my data are written. The data is written at a 45 degree angle.
